src/config/overrides/Workspaces.js

```diff
@@ -228,7 +248,32 @@ export const WORKSPACES = [
                   queryId: 'customers',
                 },
               ],
-              attributes: ['Satisfaction', 'SurroundingSatisfaction', 'Thirsty'],
+              attributes: ['Name', 'Satisfaction', 'Surrounding', 'Thirsty'],
```

Comment:
The attribute `SurroundingSatisfaction` was replaced by `Surrounding`: was it done on purpose?

Reply:
It is a mess caused by the multiple refactor of _attributes_